CONTRIBUTING.md

```diff
@@ -22,31 +24,13 @@ PRs should follow the following guidelines
 - Not be excessively large, ideally touching 100s not 1000s of lines
 - Add additional tests to ensure the correct functionality
 - Should pass all system and unit tests run on GitHub Actions CI
-- Follow the code style guidelines bellow
-
+- Follow the code style and have documentation as appropriate
 
 ### Tests
 
 Tests are located in [`tdms/tests`](./tdms/tests) and include both unit and system tests. These
 must pass locally and on CI for a change to be merged.
 
+### Developer documentation
 
-### Code style
-
-Code should follow the following guidelines
-
-- Include docstrings in header files only
-- Have docstrings in a javadoc style
-- Use "@" to define [commands](https://doxygen.nl/manual/commands.html)
-- For example:
-
-```c++
-/**
- * Add two numbers together
- *
- * @param a: real number
- * @param b: real number
- * @return sum of a and b
- */
-double add(double a, double b);
-```
+We have some more detailed developer documentation (C++ class structure, code style, etc) over [here](https://github-pages.ucl.ac.uk/TDMS) on our `gh-pages` site.
```

Comment:
Please rephrase to avoid a "[here link](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/)".

Reply:
TIL.